lib/client.js

```diff
@@ -499,14 +506,17 @@ class Parser {
       this.bufferView = new Uint8Array(this.inst.memory.buffer, this.bufferPtr, this.bufferSize)
     }
 
+    // TODO (perf): Can we avoid this copy somehow?
     this.bufferView.set(data)
```

Comment:
@mcollina Do we know any WASM expert that might be able to tell us if there is a more efficient way to pass buffers to WASM?

Reply:
Not a wasm expert, just thinking out loud. There is an API to create a shared memory:

* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory#creating_a_shared_memory

Not sure how it fits here.

Reply:
Unfortunately I don't think that will help.